scripts/codestyle.bash

```diff
@@ -31,6 +32,10 @@ ci() {
   echo "pylint"
   pylint --rcfile=.pylintrc "$BASE_PATH_DIR"/src/"$SRC_DIRECTORY_NAME" "$BASE_PATH_DIR"/tests
   echo "mypy"
+  # installing all missing stub packages (e.g. types-PyYAML, types-aiofiles, etc)
+  mkdir --parents "$CACHE_DIR"
+  mypy --cache-dir="$CACHE_DIR" --install-types
+  # runs mypy
   mypy --config-file mypy.ini --ignore-missing-imports "$BASE_PATH_DIR"/src/"$SRC_DIRECTORY_NAME" "$BASE_PATH_DIR"/tests
```

Comment:
should mypy.ini also have an absolute path (sorry I know this is not a change from this MR but it stood out)?

Reply:
done